File: test/src/main/java/com/commit451/regalia/test/TestUtil.java
package com.commit451.regalia.test;

import com.google.common.base.Charsets;
import com.google.common.io.Resources;

import java.io.IOException;
import java.net.URL;

public class TestUtil {

    public static String getCatWithListOfFriendsJson() throws IOException {
        return getFileText("cat-with-list-of-friends.json");
    }

    public static String getFileText(String filename) throws IOException {
        URL url = Resources.getResource(filename);
        return Resources.toString(url, Charsets.UTF_8);
    }
}

File: test/src/main/java/com/commit451/regalia/test/TestUtil.kt
package com.commit451.regalia.test

import java.io.IOException

object TestUtil {
    @JvmStatic
    @get:Throws(IOException::class)
    val catWithListOfFriendsJson: String
        get() = getFileText("cat_and_friends.json")

    @JvmStatic
    @Throws(IOException::class)
    fun getFileText(filename: String): String {
        // for some reason can't load from resources, so I had to improvise
        return "{\n" +
                "  \"name\": \"Lovelace\",\n" +
                "  \"friends\": [\n" +
                "    {\n" +
                "      \"name\": \"Sputnik\"\n" +
                "    },\n" +
                "    {\n" +
                "      \"name\": \"Who are you\"\n" +
                "    }\n" +
                "  ]\n" +
                "}"
    }
}